docs/ECLLanguageReference/ECLR_mods/BltInFunc-AGGREGATE.xml

```diff
@@ -93,6 +93,51 @@
             <entry>Optional. Indicates that the expression will result in more
             than 10,000 records.</entry>
           </row>
+          <row>
+            <entry><emphasis role="bold">UNORDERED</emphasis></entry>
+
+            <entry>Optional. Specifies the output record order is not significant.</entry>
+          </row>
+          <row>
+            <entry><emphasis role="bold">ORDERED</emphasis></entry>
+
+            <entry>Specifies the significance of the output record order.</entry>
+          </row>
+          <row>
+            <entry><emphasis>bool</emphasis></entry>
+
+            <entry>When False, specifies the output record order is not significant. When True, specifies the default output record order.</entry>
```

Comment:
I would have expected this to be included in the ordered entry, not as a separate one.